openedx/core/djangoapps/verified_track_content/tests/test_partition_scheme.py

```diff
@@ -143,7 +143,12 @@ def test_enrolled_in_non_selectable(self):
         self.assertEqual("Audit", self._get_user_group().name)
 
         # Now create a verified mode and check that it is returned for the learner enrolled in Credit.
-        create_mode(self.course, CourseMode.VERIFIED, "Verified Enrollment Track", min_price=1)
+        # Make the mode "expired" to ensure that credit users can still see verified-only content after
```

Comment:
instead of adding the test condition here, can we have a new test case specific for this scenario?

Reply:
Added, squashed and rebased.